web/templates/blocks/sap_instance.html.tmpl

```diff
@@ -28,8 +28,10 @@
                     <td>{{ .Elapsedtime }}</td>
                     <td>{{ .Pid }}</td>
                     <td>
-                        {{- $Textstatus := .Textstatus }}
-                        <span class='badge badge-pill badge-{{ if eq $Textstatus "Running" }}primary{{ else }}danger{{ end }}'>{{ $Textstatus }}</span>
+                        {{- $Dispstatus := .Dispstatus }}
```

Comment:
I'm not sure if we need this assignment at the beginning here. It doesn't look like improve anything